introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5-Batch-Transform.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>Do you want to write an input-output format of Batch transform? For instance, input to batch transform is a csv with number of columns ... and each row ...</p><p> </p><p>Output is blah blah..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Sure</p>